src/pages/CivicProfile.jsx

```diff
@@ -19,6 +26,8 @@ const CivicProfile = () => {
   const isSmallScreen = useMediaQuery(theme.breakpoints.down('sm'));
 
   return (
+    // TODO: Reevaluate whether we need nested Containers
+    // Containers contain styling by default. Boxes may be preferable
```

Comment:
We could try and keep the `Containers`, but I agree that `Box` would be more flexible when it comes to styling. We'll probably want to make it the preferrable component of the two.

Reply:
I decided to split the difference for now since we may want to take another look at the overall layout anyway. I think replacing this menu with tabs might be nice (as suggested in #634).